tests/download-artifacts.sh

```diff
@@ -1,16 +1,21 @@
-#!/bin/sh -eux
+#!/usr/bin/env bash
+set -o nounset -o pipefail -o errexit
+
+# Get the configuration from the current working directory.
```

Comment:
woa nice catch. do we intentionally use different versions in various tests?

Reply:
Yeah -- I initially fixed this to use the `tests/consts.sh` file everywhere and the ts-web tests failed, so I changed it to what it is now.  I think we should use the same config for everything, not sure why whoever wrote this made it working-directory-dependent.